content/projects/gsoc/students.adoc

```diff
@@ -50,31 +81,203 @@ We highly recommend to make some contributions to the project while you work on
 It will help you to polish the proposal,
 and mentors will consider contributions and interactions with the community when processing applications.
 
-Useful links:
+[[UsefulLinks]]
+=== Useful links
+
+Here are a list of links to help you get started on participating in Jenkins and in coding for the Jenkins project,
+in increasing level of difficulty.
+
+* link:https://jenkins.io/project/governance/[Project Governance]
+* link:https://github.com/jenkinsci/jenkins/blob/master/CONTRIBUTING.md[Contributing to Jenkins] (for contributions to the core)
```

Comment:
Not sure it is the best link to start from

Reply:
Do you have another link to suggest? I moved it below a different title.